doc/deploy/android.md

```diff
@@ -1,33 +1,129 @@
-## Deploying to the Google Play Store
-##### Prerequirements
-- You **must** read the [general deploy info](README.md).
-- You **must** have the official PVTrAck signing key. See [the keys doc](android-keys.md) for more info.
-- You **must** have the Android SDK and Android build tools installed (both bundled with Android Studio). We target API level 25 (Nougat, 7.1).
-- You **must** have access to the [Google Play Developer Console](play.google.com/apps/publish) (see @sherson).
-- You *should* have [Android Studio](https://developer.android.com/studio/index.html) installed.
+# Building an Android App to Deploy in Google Play
 
-##### Steps
+## Requirements
```

Comment:
Worth mentioning, also, that almost all of these requirements are documented here:

https://cordova.apache.org/docs/en/latest/guide/platforms/android/index.html

Reply:
I think it might be worth just linking to those instructions and these instructions:

https://cordova.apache.org/docs/en/latest/guide/platforms/ios/index.html

In the main readme rather than duplicating them here.